src/site/content/en/blog/autowebperf/index.md

```diff
@@ -6,6 +6,12 @@ description: >
   A new modular tool that enables automatic gathering of performance data form multiple sources.
 date: 2020-12-09
 # updated: 2020-07-22
+authors:
+  - jonchen
+  - gilbertococchi
+  - antoinebisch
+hero: hero.png
+thumbnail: hero.png
```

Comment:
```suggestion
```
There doesn't appear to be a thumbnail in this PR. Did you meant to add one?

Reply:
Yes. Should I upload another low-res image for thumbnail?

Reply:
Here is updated guidance on how to use `thumbnail`: https://deploy-preview-4359--web-dev-staging.netlify.app/handbook/markup-media/#thumbnail